### DISCUSSION TITLE
if ceph is deployed by cephadm，not rook，how to backup and restore ceph

### DISCUSSION THREAD

**Discussion by 13567436138** [rook/rook#13598] [Q&A] [answered]
if ceph is deployed by cephadm，not rook，how to backup and restore ceph,sorrry to ask question here ,as ceph does not have issue or discussion

**Comment by phlogistonjohn** [comment, marked as answer]
Sorry for not answering the question but I wanted to share the following reaction to `as ceph does not have issue or discussion` - Ceph upstream has a number of channels for feedback available, though they are not on the github platform:

There's a round up page here: https://ceph.io/en/community/connect/
Ceph issues go to the tracker, here: https://tracker.ceph.com/

For general discussion you're best off using the mailing list. There's a web interface here https://lists.ceph.io/hyperkitty/list/ceph-users@ceph.io/
There's also discussions on slack (and via a bridge to irc): https://join.slack.com/t/ceph-storage/shared_invite/zt-1s4wudrqa-Ak6dxRKdvLoSTKxpAvaJQg  then join the `#ceph` channel.

(Edit: corrected link to ceph-users list)
